plugins/defaultplugins/ifplugin/interface_config.go

```diff
@@ -571,6 +629,15 @@ func (plugin *InterfaceConfigurator) deleteVPPInterface(oldConfig *intf.Interfac
 	}
 
 	// let's try to do following even if previously error occurred
+	addr, isIpv6, err := addrs.ParseIPWithPrefix(oldConfig.ContainerIpAddress)
```

Comment:
do we need to do this if the interface will be removed?

Reply:
I do not know where this setting (container IP address) is stored in VPP (whether it is part of interface configuration).
I would say that it isn't necessary to have it here, but I am not 100 % sure.